### PR TITLE
Correct possible code errors for example 7

### PR DESCRIPTION
**Pull request by darren2pro**
#### Summary

There was a mild syntax error when I tested the code for example 7 in this page. The bugs and my fixes are described below.

- Bug1: Re-declare the variable `e`
    - Fix: Change the variable name to `event`
- Bug2: Assign to constant variable `row`
    - Fix: Change the declaration keyword to `var`
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I agree that we should use `event` instead of `e` (as `e` is the parameter name), but `row` seems correct to me.

File: files/en-us/web/api/document_object_model/examples/index.md [hunk 1]
```diff
@@ -303,12 +303,12 @@ function showEventProperties(e) {
...
 
-  const e = e || window.event;
-  document.getElementById('eventType').innerHTML = e.type;
+  const event = e || window.event;
+  document.getElementById('eventType').innerHTML = event.type;
 
   const table = document.createElement('table');
   const thead = table.createTHead();
-  const row = thead.insertRow(-1);
+  var row = thead.insertRow(-1);
```
[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/web/api/document_object_model/examples/index.md:311]: I didn't see line 323, `let` is fine then.

[the remark this answers is not in the document]
Comment (darren2pro) [reply files/en-us/web/api/document_object_model/examples/index.md]: @teoli2003 done! Thank you

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**